Lib/test/test_interpreters.py

```diff
@@ -463,6 +464,27 @@ def test_bytes_for_script(self):
     # test_xxsubinterpreters covers the remaining Interpreter.run() behavior.
 
 
+class StressTests(TestBase):
+
+    # In these tests we generally want a lot of interpreters,
+    # but not so many that any test takes too long.
+
+    def test_create_many_sequential(self):
+        alive = []
+        for _ in range(100):
+            interp = interpreters.create()
+            alive.append(interp)
+
+    def test_create_many_threaded(self):
```

Comment:
```suggestion
    @requires_resource('cpu')
    def test_create_many_threaded(self):
```

Reply:
Hmm, what is the effect of this decorator?  Also, what is it telling readers?

I wouldn't consider this test to be particularly CPU-intensive.  We do spin up a couple hundred threads but not so many that this test should take more than a few seconds on a resource-starved host.  Likewise, this shouldn't impose a significant memory burden.  (Each interpreter uses ~3kB.)

Reply:
https://github.com/python/cpython/blob/cdeb1a6caad5e3067f01d6058238803b8517f9de/Lib/test/libregrtest/cmdline.py#L108

AFAIK, Conventionally, most CPU-intensive tests consume a few seconds in high-performance computers, we added the flag for low-performance CPUs to skip this test.

I thought that this test look quite CPU-intensive test, so I suggested adding it.